src/coreclr/tools/Common/TypeSystem/Mutable/MutableModule.Sorting.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Threading;
+
+using Debug = System.Diagnostics.Debug;
+
+namespace Internal.TypeSystem.Ecma
+{
+    partial class MutableModule
+    {
+        // This isn't deterministic, but it is functional.
```

Comment:
Am I right to understand this actually doesn't violate multithreaded Crossgen2 determinism because in practice there's always at most one mutable module per compilation?

Reply:
Yes. I've updated the comment.